.github/workflows/pr_title.yml

```diff
@@ -0,0 +1,27 @@
+name: Pull Request
+
+on:
+  pull_request_target:
+    types: ['opened', 'reopened', 'edited', 'synchronize']
+
+jobs:
+  faency:
+    name: Lint PR Title
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*'
+
+      - name: Install dependencies
+        run: npm ci
```

Comment:
>  ❓Question

Do we need to install deps to do the title check?

Reply:
No longer , good call!